website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -139,23 +139,23 @@ This POST endpoint requires a JSON object with the following properties as a req
+| `data.attributes.refresh-only`                     | bool                 | false                                              | Whether this run should refresh the state without modifying any resources. Mutually exclusive with `is-destroy`.                                                                                                                                                                                                                                                                                                              |
+| `data.attributes.replace-addrs`                    | array\[string]       |                                                    | Specifies an optional list of resource addresses to be passed to the `-replace` flag.                                                                                                                                                                                                                                                                                                                                         |
+| `data.attributes.target-addrs`                     | array\[string]       |                                                    | Specifies an optional list of resource addresses to be passed to the `-target` flag.                                                                                                                                                                                                                                                                                                                                          |
+| `data.attributes.variables`                        | array\[{key, value}] | (empty array)                                      | Specifies an optional list of run-specific variable values. Refer to [Run-Specific Variables](/terraform/cloud-docs/workspaces/variables/managing-variables#run-specific-variables) for details.                                                                                                                                                                                                                              |
+| `data.attributes.plan-only`                        | bool                 | (from configuration version)                       | Specifies if this is a [speculative, plan-only](/terraform/cloud-docs/run/modes-and-options#plan-only-speculative-plan) run that Terraform cannot apply. Often used in conjunction with terraform-version in order to test whether an upgrade would succeed.                                                                                                                                                                  |
```

Comment:
```suggestion
| `data.attributes.plan-only`                        | bool                 |  Depends on configuration version | Specifies if this is a [speculative, plan-only](/terraform/cloud-docs/run/modes-and-options#plan-only-speculative-plan) run that Terraform cannot apply. Often used in conjunction with terraform-version in order to test whether an upgrade would succeed.                                                                                                                                                                  |
```
I'm not sure, but I'm interpreting that "(from configuration version)" means that you can specify the plan type in the configuration and that each version of the configuration can have different types.

Reply:
I believe "from configuration version" here means that the default value is set by _your_ configuration version, not that the behavior is different depending on the version. So, I'm going to leave as is.